components/Menu/structure.ts

```diff
@@ -110,6 +110,23 @@ const menu: MenuCategoryProps[] = [
           },
         ],
       },
+      {
+        title: "By Compliance",
```

Comment:
```suggestion
        title: "By Compliance Standard",
```
"By Compliance" sounded odd to me, since, for example, FedRAMP is not a compliance, but a compliance standard.